backend/pkg/database/database.go

```diff
@@ -314,7 +315,18 @@ GROUP BY vulnerabilities.id,
          p.name,
          p.version,
          p.id,
-         pv.fix_version
+         pv.fix_version;
+  `
+
+	cisDockerBenchmarkResultsViewQuery = `
+CREATE VIEW cis_d_b_checks_view AS
+SELECT resource_cis_d_b_checks.*,
```

Comment:
do you need all `resource_cis_d_b_checks` fields or only resource id?

Reply:
No, you are right. Thx